지미_김지명/2주차_미션/mission2/src/useContext/Navbar.tsx

```diff
@@ -0,0 +1,19 @@
+import { THEME, useTheme } from "../context/ThemeProvider"
+import ThemeToggleButton from "./ThemeToggleButton";
+import clsx from "clsx";
+
+export default function Navbar() {
+
+  const { theme, toggleTheme }= useTheme();
+
+  const isLightMode = theme === THEME.LIGHT;
+
+  return (
+    <nav className= {clsx (
```

Comment:
clsx를 사용해도 되지만 백틱(`)을 사용하여 개발할 수도 있습니다!


```suggestion
    <nav className={`p-4 w-full flex justify-end ${isLightMode ? 'bg-white':'bg-gray-800'}`} ></nav>
```